config/values/00-values.yml

```diff
@@ -11,6 +11,10 @@ cf_admin_password:
 #! reserved static ip for istio LoadBalancer
 istio_static_ip: ""
 
+#! for highly available deployments, increase to 2
+istiod:
```

Comment:
should we put all istio configs under one field called `istio`?

Reply:
Yeah, that sounds good to us. For `istio_static_ip`, we can keep that one separate and we'll plan on renaming that key at a later data ([story](https://www.pivotaltracker.com/story/show/174852214))

Reply:
But given our primary review comment here, we're requesting that you not include the configurability as part of this PR - please remove the added data value